### PR TITLE
Add util to remove high-cadence data

### PR DESCRIPTION
**Pull request by bfhealy**
This PR adds a function to `scope/utils.py` to remove all but the first high-cadence data points from a light curve.

### CODE REVIEW

File: scope/utils.py [hunk 1]
```diff
@@ -635,6 +635,20 @@ def overlapping_histogram(a, bins):
     return n, (bins[:, 0] + bins[:, 1]) / 2.0
 
 
+def removeHighCadence(t, m, e, cadence_minutes=30.0):
+    idx = []
+    for ii, _ in enumerate(t):
```
Comment (mcoughlin) [inline scope/utils.py:640]: Probably don't need enumerate here.

**Review by mcoughlin — APPROVED** [review]
Small comment.

File: scope/utils.py (hunk 1 above)
Comment (bfhealy) [reply scope/utils.py]: Agreed and replaced.

Comment (mcoughlin) [reply scope/utils.py]: Should be good to merge!

**Pull request merged**